app/Console/Commands/InteractsWithCollections.php

```diff
@@ -34,6 +36,7 @@ public function forEachCollection(Closure $callback, Closure $queryCallback = nu
             ->when($queryCallback !== null, $queryCallback)
             ->select('collections.*')
             ->withoutSpamContracts()
+            ->where('type', TokenType::Erc721)
```

Comment:
I would add a custom scope like scopeErc721() so this can be changed to `->erc721(`

just a suggestion feel free to ignore